app/controllers/observations_controller/index.rb

```diff
@@ -183,7 +183,7 @@ def define_index_args(query, args)
     args = { controller: "/observations",
              action: :index,
              matrix: true,
-             include: [:name, :location, :user, :rss_log,
+             include: [:name, :location, :user, :namings, :rss_log,
                        { thumb_image: :image_votes }] }.merge(args)
 
```

Comment:
Note the new `includes(:namings)` requested by Bullet here.

Reply:
Removed in [4880aa6](https://github.com/MushroomObserver/mushroom-observer/pull/1350/commits/4880aa6330fd93872e6df1afcd6fdae168e98772)